engines/bops_api/app/views/bops_api/v2/public/search.json.jbuilder

```diff
@@ -12,3 +12,5 @@ end
 json.data @planning_applications.each do |planning_application|
   json.partial! "show", planning_application:
 end
+
+json.links pagy_jsonapi_links(@pagy)
```

Comment:
nitpick but do we want to put this at the top of the response?

Reply:
I'm not sure whether it'll matter but it certainly can't hurt.